SKYKitChat/Classes/Core/Cache/SKYChatCacheController.h

```diff
@@ -33,6 +33,12 @@
                                   order:(NSString *)order
                              completion:(SKYChatFetchMessagesListCompletion)completion;
 
+- (void)fetchMessagesWithConversationID:(NSString *)conversationId
+                                  limit:(NSInteger)limit
+                        beforeMessageId:(NSString *)beforeMessageId
```

Comment:
`beforeMessageID` seems better

Reply:
ok, but it sucks conversationId and conversationID co-exist all over the place.